src/ripple/app/tx/Transaction.cpp

```diff
@@ -107,6 +107,24 @@ void Transaction::setStatus (TransStatus ts, std::uint32_t lseq)
     mInLedger   = lseq;
 }
 
+TransStatus Transaction::sqlTransactionStatus(
+    boost::optional<std::string> const& status)
+{
+    char const c = (status) ? (*status)[0] : TXN_SQL_UNKNOWN;
+
+    switch (c)
+    {
+    case TXN_SQL_NEW:       return NEW;
```

Comment:
I wanted it to be very clear that we're returning and not breaking. Aligning the `return` statements seemed to help. I'm open to alternatives.